token-lending/program/tests/helpers/mod.rs

```diff
@@ -245,7 +260,7 @@ pub fn add_reserve(
 
     let reserve_keypair = Keypair::new();
     let reserve_pubkey = reserve_keypair.pubkey();
-    let mut reserve_state = ReserveState::new(1, liquidity_amount);
+    let mut reserve_state = ReserveState::new(1u64.wrapping_sub(slots_elapsed), liquidity_amount);
```

Comment:
It's a hack to accrue interest. I couldn't think of another way to mock out the clock sysvar for integration tests. Integration tests currently always run at slot 1 so the wrapping sub will set the last_update_slot to a very high number so that `1 - last_update_slot` is equal to my desired `slots_elapsed`

Reply:
Gotcha, I assumed that `1 - last_update_slot` would panic in that case, since that's the normal rust behavior.  Does the VM silently wrap around all overflows and underflows?

Reply:
Rust actually doesn't panic in release mode on overflows/underflows, only in debug mode (I just recently learned this, so happy to share). So, what I believe is happening is that the rust compiler in release mode is emitting MIR that doesn't have overflow/underflow checks and that gets translated directly to BPF instructions. The BPF VM does not check for arithmetic overflows/underflows.

Reply:
Gotcha, thanks for sharing!